packaging/installer/netdata-updater.sh

```diff
@@ -369,6 +369,167 @@ update() {
   return 0
 }
 
+update_static() {
+  ndtmpdir="$(create_tmp_directory)"
+  PREVDIR="$(pwd)"
+
+  echo >&2 "Entering ${ndtmpdir}"
+  cd "${ndtmpdir}" || exit 1
+
+  sysarch="$(uname -m)"
+
+  download "${NETDATA_TARBALL_CHECKSUM_URL}" "${ndtmpdir}/sha256sum.txt"
+  download "${NETDATA_TARBALL_URL}" "${ndtmpdir}/netdata-${sysarch}-latest.gz.run"
+  if ! grep netdata-latest.gz.run "${ndtmpdir}/sha256sum.txt" | safe_sha256sum -c - > /dev/null 2>&1; then
+    fatal "Static binary checksum validation failed. Stopping netdata installation and leaving binary in ${ndtmpdir}\nUsually this is a result of an older copy of the file being cached somewhere and can be resolved by simply retrying in an hour."
+  fi
+
+  if [ -e /opt/netdata/etc/netdata/.install-type ] ; then
+    install_type="$(cat /opt/netdata/etc/netdata/.install-type)"
+  else
+    install_type="INSTALL_TYPE='legacy-static'"
+  fi
+
+  # Do not pass any options other than the accept, for now
+  # shellcheck disable=SC2086
+  if sh "${ndtmpdir}/netdata-${sysarch}-latest.gz.run" --accept -- ${REINSTALL_OPTIONS}; then
+    rm -r "${ndtmpdir}"
+  else
+    echo >&2 "NOTE: did not remove: ${ndtmpdir}"
+  fi
+
+  echo "${install_type}" > /opt/netdata/etc/netdata/.install-type
+
+  if [ -e "${PREVDIR}" ]; then
+    echo >&2 "Switching back to ${PREVDIR}"
+    cd "${PREVDIR}"
+  fi
+}
+
+update_binpkg() {
+  os_release_file=
+  if [ -s "/etc/os-release" ] && [ -r "/etc/os-release" ]; then
+    os_release_file="/etc/os-release"
+  elif [ -s "/usr/lib/os-release" ] && [ -r "/usr/lib/os-release" ]; then
+    os_release_file="/usr/lib/os-release"
+  else
+    fatal "Cannot find an os-release file ..." F0401
+  fi
+
+  # shellcheck disable=SC1090
+  . "${os_release_file}"
+
+  DISTRO="${ID}"
+
+  supported_compat_names="debian ubuntu centos fedora opensuse"
+
+  if str_in_list "${DISTRO}" "${supported_compat_names}"; then
```

Comment:
Where is `str_in_list` defined? Same for `INTERACTIVE` below. If we expect these to come from env then list them explicitly in the 1st comment of this file.

Reply:
Both are missing definitions that I forgot to deal with when copying some of this code over from kickstart-ng.sh.